binstar_client/__init__.py

```diff
@@ -57,6 +57,8 @@ def __init__(self, token=None, domain='https://api.anaconda.org', verify=True, *
 
         if domain.endswith('/'):
             domain = domain[:-1]
+        if not domain.startswith(('http://', 'https://')):
```

Comment:
Actually, the idea was replace urls without `http://` **or** `https://`, not to force `https://` on every domain